Gemfile.lock

```diff
@@ -138,17 +138,17 @@ GEM
       sshkit-interactive (~> 0.3.0)
     case_transform (0.2)
       activesupport
-    code_analyzer (0.4.8)
+    code_analyzer (0.5.1)
```

Comment:
### [code_analyzer](https://github.com/flyerhzm/code_analyzer)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.4.8 → 0.5.1

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **6** commits. See the full changes on [the compare page](https://github.com/flyerhzm/code_analyzer/compare/v0.4.8...v0.5.1).

These are the individual commits:
- (c15155a) [Auto corrected by following Format Ruby Code](https://github.com/flyerhzm/code_analyzer/commit/c15155a85d7ec340471a0600745204b016d7e0a8)
- (687afd3) [fix after upgrading to ruby 2.6](https://github.com/flyerhzm/code_analyzer/commit/687afd39695de7a09f02331a3869414e867784e5)
- (f4990d6) [Bumping version to 0.5.0](https://github.com/flyerhzm/code_analyzer/commit/f4990d63958d3319f43b97309b4962503f9abe05)
- (04fd574) [remove 1.9.3 compatible tests](https://github.com/flyerhzm/code_analyzer/commit/04fd574f0966470afe0f14f6a3028e6fd043da58)
- (657c93a) [support :@period for to_s](https://github.com/flyerhzm/code_analyzer/commit/657c93a441478e49846fd62cbfac20c8b59a0941)
- (267a4a3) [Bumping version to 0.5.1](https://github.com/flyerhzm/code_analyzer/commit/267a4a3e1da22771d447b8bfe108cfdaa5869aa8)


</details>